src/AppContainer.tsx

```diff
@@ -50,21 +54,6 @@ const ToolBarContent: FunctionComponent<ToolBarContentProps> = ({ workspaceInfo,
 }
 //////////////////////////////////////////////////////////////
 
-const SetWorkspaceInfo: FunctionComponent<{workspaceName: string | null, feedUri: string | null, onSetWorkspaceInfo: (di: WorkspaceInfo) => void}> = ({ workspaceName, feedUri, onSetWorkspaceInfo }) => {
```

Comment:
This mechanism for setting the workspace info was always VERY awkward. Glad it's gone now. The workspaceInfo is no longer part of the redux state.